package.json

```diff
@@ -40,5 +40,8 @@
   },
   "engines": {
     "node": ">=12.x"
+  },
+  "dependencies": {
+    "mongoose": "^5.12.14"
```

Comment:
I think you could remove mongoose from dependencies in root project as it is specifically used in the `backend` packages

Reply:
just remove the dependenciy? will do

Reply:
yes, or you can use `yarn remove mongoose` from project root folder

Reply:
done